src/components/views/dialogs/FeedbackDialog.tsx

```diff
@@ -19,30 +19,33 @@ import QuestionDialog from './QuestionDialog';
 import { _t } from '../../../languageHandler';
 import Field from "../elements/Field";
 import AccessibleButton from "../elements/AccessibleButton";
-import CountlyAnalytics from "../../../CountlyAnalytics";
+import CountlyAnalytics, { Rating } from "../../../CountlyAnalytics";
 import SdkConfig from "../../../SdkConfig";
 import Modal from "../../../Modal";
 import BugReportDialog from "./BugReportDialog";
 import InfoDialog from "./InfoDialog";
 import StyledRadioGroup from "../elements/StyledRadioGroup";
+import { IDialogProps } from "./IDialogProps";
 
 const existingIssuesUrl = "https://github.com/vector-im/element-web/issues" +
     "?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc";
 const newIssueUrl = "https://github.com/vector-im/element-web/issues/new/choose";
 
-export default (props) => {
-    const [rating, setRating] = useState("");
-    const [comment, setComment] = useState("");
+interface IProps extends IDialogProps {}
 
-    const onDebugLogsLinkClick = () => {
+const FeedbackDialog: React.FC<IProps> = (props: IProps) => {
+    const [rating, setRating] = useState<string>();
```

Comment:
should this not be `Rating`?

Reply:
That seems to make `StyledRadioGroup` very unhappy about types :(

Reply:
then you should be able to do `value={String(rating)}` to appease it, or even `value={rating as number}`